packages/gp2-components/src/molecules/UserCardInfo.tsx

```diff
@@ -3,9 +3,9 @@ import { crossQuery, Link, pixels } from '@asap-hub/react-components';
 import { gp2 as gp2Routing } from '@asap-hub/routing';
 import { css } from '@emotion/react';
 import { ComponentProps, Fragment } from 'react';
-import ProjectIcon from '../icons/project-icon';
+import projectIcon from '../icons/project-icon';
```

Comment:
can we group this icon with workingGroupIcon
```suggestion
import { projectIcon, workingGroupIcon } from '../icons';
```